schemas/kits-schema.json

```diff
@@ -45,7 +45,11 @@
             },
             "visualStudio": {
                 "type": "string",
-                "description": "Name of the Visual Studio product"
+                "description": "Instance ID of the Visual Studio product"
+            },
+            "visualStudioVersion": {
```

Comment:
Changing kits is not recommended if we can manage it.  Can this information be supplied some other way, or be internal for when we hydrate the Kit objects?  I thought that Kits wouldn't be used for Presets.

Reply:
Kits are reused as storage for compiler information. It's an optional field, so it's compatible with the previous scans, and therefore rescan is not required if the users don't need some of the new features. So I think it's acceptable to make this change.

The other option we have is to invoke and parse vswhere by calling `vsInstallations()` in visual-studio.ts, and then iterate through all the instances for each VS kit to look for the versions. This feels like a less ideal way to me.

Reply:
Sounds good I'll change this.